workbench/src/main/main.js

```diff
@@ -184,17 +195,20 @@ export function main() {
       createWindow();
     }
   });
-  app.on('window-all-closed', async () => {
+  app.on('window-all-closed', async (event) => {
     // On OS X it is common for applications and their menu bar
     // to stay active until the user quits explicitly with Cmd + Q
+    event.preventDefault();
```

Comment:
The default behavior of the [`window-all-closed` event](https://www.electronjs.org/docs/latest/api/app#event-window-all-closed) is to quit the app, so the intent is to prevent quitting on macOS. You made me realize it makes more sense to write it this way:
```
  app.on('window-all-closed', (event) => {
    if (process.platform === 'darwin') {
      event.preventDefault();
    }
  });
```

Reply:
Okay, thanks @emlys . I think the docs are slightly unclear here. One way to read it is that simply subscribing to this event already prevents the default behavior of quitting. And that seems like the way it was working previously, where closing the window did not quit on darwin.

With this configuration closing the window no longer quits the app on Windows. So that makes me think that subscribing to this event already prevents the default behavior. I think we can just revert this back to: ?

```
  app.on('window-all-closed', () => {
    // On OS X it is common for applications and their menu bar
    // to stay active until the user quits explicitly with Cmd + Q
    if (process.platform !== 'darwin') {
      app.quit();
    }
  });
```

Reply:
I think the `window-all-closed` handler might be unnecessary now, since on mac, we minimize the window instead of closing it. And it seems like we want the default behavior on Windows.